Content/GUI/PlayerStats/PlayerStatInnerPanel.cs

```diff
@@ -15,8 +15,18 @@ internal class PlayerStatInnerPanel : SmartUIElement
 	private UICharacter _drawDummy = null;
 	private int _offset = 0;
 
+	public override void SafeMouseOver(UIMouseEvent evt)
+	{
+		Main.blockMouse = true;
+		Main.isMouseLeftConsumedByUI = true;
+		Main.LocalPlayer.mouseInterface = true;
+
+		Main.NewText("egg");
```

Comment:
Same

Reply:
You have to make debugging fun somehow (whoops)